readme.md

```diff
@@ -124,7 +140,89 @@ There is no default export.
 
 Default schema ([`Schema`][api-schema]).
 
-Follows [GitHub][] style sanitation.
+Follows [GitHub][] style sanitation. 
+
+#### Overriding the default schema
+
+In some cases, it may be necessary to override the default schema.
+In the example below, the `id` attribute of the `<h2>` tag is removed 
+while the `<h1>` tag is unaltered.
+
+```js
+import rehypeParse from "rehype-parse"
+import rehypeSanitize from "rehype-sanitize"
+import rehypeStringify from "rehype-stringify"
+import { unified } from "unified"
+
+const file = await unified()
+  .use(rehypeParse)
+  .use(rehypeSanitize)
+  .use(rehypeStringify)
+  .process(
+    `<h1 id="bohemian">I'm just a poor boy</h1>
+<h2 id="rhapsody">From a poor family</h2>`
+  )
+
+console.log(String(file))
+
+// === Output ========
+// <h1 id="user-content-bohemian">I'm just a poor boy</h1>
+// <h2>From a poor family</h2>
+```
+
+This happens because the 
+[default schema](https://github.com/syntax-tree/hast-util-sanitize/blob/main/lib/schema.js) 
+only allows `id` equal to `'footnote-label'` for `<h2>` tags.
+
+```js
+// lib/schema.js
+
+export const defaultSchema = {
+  ancestors: {...},
+  attributes: {
+  ...
+  h2: [
+        ['id', 'footnote-label'],
+        ['className', 'sr-only'],
+      ],
+  ...
+  }
+}
```

Comment:
I’d like to actually fix this bug. So I’d prefer it if this example was unrelated to `h2` and `className`.